frontend/src/core/comments/components/CommentsList.css

```diff
@@ -1,12 +1,19 @@
 .comments-list {
     line-height: 22px;
-    padding: 8px;
 }
 
 .comments-list .comment {
     display: flex;
     margin-left: 58px;
 }
+.comments-list ul li {
+    border-bottom: 1px solid #5e6475;
+    padding: 8px;
```

Comment:
This will add 16px padding between comments (8px above and 8px below).

Reply:
And we also need to add left and right padding to the add comment form.

Reply:
For the padding did you want to not have that much padding between the comments?

Reply:
No, I'd like it to remain the same as on master.